beacon_node/http_api/src/lib.rs

```diff
@@ -1696,8 +1697,12 @@ pub fn serve<T: BeaconChainTypes>(
                                     .to_execution_address;
 
                                 // New to P2P *and* op pool, gossip immediately if post-Capella.
-                                let publish = chain.current_slot_is_post_capella().unwrap_or(false);
-                                if publish {
+                                let capella_broadcast = if chain.current_slot_is_post_capella().unwrap_or(false) {
+                                    CapellaBroadcast::No
+                                } else {
+                                    CapellaBroadcast::Yes
+                                };
```

Comment:
Won't this always be `CapellaBroadcast::No` for messages received prior to the fork? Which will result in no changes being broadcast with a delay?